### DISCUSSION TITLE
The kernel lacks nfs v3 support

### DISCUSSION THREAD

**Discussion by pvervoo** [ltsp/ltsp#949] [General]
Hi

I installed LTSP chrootless on a debian 12 server.  When booting the client, I get the message that the nfs mount failed:
Begin: Retrying nfs mount ... mount: the kernel lacks NFS v3 support
done.
This message is repeated a number of times, then eventually I get the message:
Begin: Running /scripts/nfs-bottom ... done.
Begin: Running /scripts/init-bottom ... Running /sbin/ltsp initrd-bottom
Image doesn't exist: /root/images/x86_64.img
LTSP command failed: mount_img_src /root/images/x86_64.img /root /run/initramfs/ltsp
Stopping plymouth
Aborting ltsp
LTSP boot error!  Enable DEBUG_SHELL to troubleshoot!

On my server NFS is running and has v3 and v4 enabled.

What is going wrong ?

**Comment by alkisg** [comment]
This isn't a bug in LTSP; converting into a discussion.

**Comment by alkisg** [comment]
This probably means that you have specified the wrong kernel command line parameters.
We don't want the kernel to mount anything, we want the initramfs to do that, using nfsmount.
Please use the LTSP installation instructions.
